Source/FlowEditor/Private/DetailCustomizations/FlowNode_CustomInputDetails.cpp

```diff
@@ -1,101 +1,35 @@
 // Copyright https://github.com/MothCocoon/FlowGraph/graphs/contributors
 
 #include "DetailCustomizations/FlowNode_CustomInputDetails.h"
+#include "DetailLayoutBuilder.h"
 #include "FlowAsset.h"
-#include "Nodes/Route/FlowNode_CustomInput.h"
-
-#include "DetailCategoryBuilder.h"
-#include "DetailWidgetRow.h"
-#include "PropertyEditing.h"
-#include "Widgets/Input/SComboBox.h"
-#include "Widgets/Text/STextBlock.h"
 
 #define LOCTEXT_NAMESPACE "FlowNode_CustomInputDetails"
 
-void FFlowNode_CustomInputDetails::CustomizeDetails(IDetailLayoutBuilder& DetailLayout)
+FFlowNode_CustomInputDetails::FFlowNode_CustomInputDetails()
 {
-	DetailLayout.GetObjectsBeingCustomized(ObjectsBeingEdited);
```

Comment:
_For best diff, compare FlowNode_CustomInputDetails.h/cpp vs. FlowNode_CustomNodeBase.h/cpp.

This is the most consequential code change in the changelist.  It restructures how some of the Details code works/stores its data to fix a bug with the ComboBox list contents._